docs/pages/includes/tls-multiplexing-warnings.mdx

```diff
@@ -1,17 +1,35 @@
-Prior to Teleport `13.0`, using Teleport's TLS routing mode behind a layer 7 (HTTP/HTTPS) proxy is generally not supported, due to
-these proxies terminating TLS themselves and then rewriting their requests to the upstream service, stripping
-the additional SNI/ALPN parts of the request in the process.
+#### Teleport version 13.0+
 
 Support for TLS routing behind layer 7 (HTTP/HTTPS) load balancers and reverse
 proxies is available starting from Teleport `13.0`. Please ensure your Teleport
 cluster and Teleport clients are up to date. If the problem persists, please
 submit a [GitHub
 issue](https://github.com/gravitational/teleport/issues/new/choose).
 
+You must ensure that your reverse proxy is communicating with Teleport using HTTPS.
+When running Teleport in Kubernetes and using nginx as an ingress, this requires adding
+an annotation to the chart values:
+
+```yaml
+annotations:
+  ingress:
+    nginx.ingress.kubernetes.io/backend-protocol: "HTTPS"
+```
+
+Deploying Teleport behind Cloudflare, whether using its proxy ("orange-clouding") or tunnels
+(`cloudflared`) should work with Teleport version 15.1 or higher. See the [TLS Routing FAQ]
+(../../architecture/tls-routing/#will-tls-routing-work-behind-my-reverse-proxy) for more details.
+
+#### Prior to Teleport version 13.0
+
+Prior to Teleport version `13.0`, using Teleport's TLS routing mode behind a layer 7 (HTTP/HTTPS) proxy is not supported,
```

Comment:
```suggestion
Prior to Teleport version 13.0, using Teleport's TLS routing mode behind a layer 7 (HTTP/HTTPS) proxy is not supported,
```

No need for the backticks here. Let's just refer to a version number like any other text (and just like you did with 15.1 on Line 20)